### ISSUE TITLE
windows builds are failing?

### ISSUE THREAD

**Issue by colinxs** [Lyceum/LyceumDocs.jl#17]

**Comment by klowrey** [comment]
Linux builds are also failing; seems like adding in LyceumMuJoCoViz depends on GLFW which has problems when headless.

**Comment by colinxs** [comment]
yeah looks like I'll have to do the same tricks I did for LyceumMuJoCoViz's CI
